testing/python/issue/test_tilelang_lop3_i2s_to_i4s_decode.py

```
@@ -0,0 +1,84 @@
+        out_dtype=T.int4,
+        source_format=source_format,
+        source_bit=2,
+        storage_dtype=T.int8,
+        with_scaling=False,
+        with_zeros=False,
+    )
+    source = lop3["c_source"]
+    func_name = lop3["func_name"]
+
+    @T.prim_func
+    def main(I: T.Tensor((I2_STORAGE_BYTES,), T.int8), O: T.Tensor((I4_STORAGE_BYTES,), T.int8)):
```

Ruff (0.15.20)

[error] Ambiguous variable name: `I`

(E741)